hudi-utilities/src/test/java/org/apache/hudi/utilities/sources/TestGcsEventsSource.java

```diff
@@ -63,14 +60,8 @@ public static void beforeAll() throws Exception {
     UtilitiesTestBase.initTestServices(false, false);
   }
 
-  @AfterAll
-  public static void afterAll() {
```

Comment:
These parent methods were all being called twice since the test is extending UtilitiesTestBase